scripts/generate_schemas.py

```diff
@@ -277,9 +297,35 @@ def convertAvro(self, avdlFile):
             print("converting", avdlFile)
         if self.verbosity > 1:
             print("running:", " ".join(args))
-        # TODO silence the Java output unless we are in verbose mode by
-        # capturing stdout/stderr here
-        subprocess.check_call(args)
+        if self.verbosity > 1:
+            subprocess.check_call(args)
+        else:
+            devnull = open(os.devnull, 'w')
```

Comment:
Using with instead of open/close should probably also be a directive in the future style guide; saves a line of code and is more robust to boot.